seqr/views/utils/variant_utils.py

```diff
@@ -27,13 +29,53 @@
 OMIM_GENOME_VERSION = GENOME_VERSION_GRCh38
 
 
-def update_project_saved_variant_json(project, family_id=None, user=None):
-    saved_variants = SavedVariant.objects.filter(family__project=project).select_related('family')
-    if family_id:
-        saved_variants = saved_variants.filter(family__family_id=family_id)
+def update_projects_saved_variant_json(projects, user_email, **kwargs):
+    success = {}
+    skipped = {}
+    error = {}
+    logger.info(f'Reloading saved variants in {len(projects)} projects')
+    for project_id, project_name, family_ids in tqdm(projects, unit=' project'):
+        try:
+            updated_saved_variant_guids = update_project_saved_variant_json(
```

Comment:
Just to make sure I've followed this correctly, this makes a single search call to the hail search service per project, with all of the saved variants embedded in the POST body?

Reply:
yes, although if theres more than 1000 variants it does them in batches

Reply:
There is a better way to do it but it involves some changes to the hail backend, so I was going ot do that as a separate PR: https://github.com/broadinstitute/seqr-private/issues/1362#issuecomment-1971344093